src/commcare_cloud/ansible/roles/supervisor/templates/supervisord.conf.j2

```diff
@@ -21,7 +21,7 @@ password={{ supervisor_http_password }}               ; (default is no password
 [supervisord]
 logfile=/tmp/supervisord.log ; (main log file;default $CWD/supervisord.log)
 logfile_maxbytes=20MB        ; (max main logfile bytes b4 rotation;default 50MB)
-logfile_backups=10           ; (num of main logfile rotation backups;default 10)
+logfile_backups=5            ; (num of main logfile rotation backups;default 10)
```

Comment:
fyi This is going to restart supervisor, and I've had issues with that leaving zombie processes around before.

Reply:
The other option is to make it configurable per env. I've already rolled this out to swiss - but that also included a supervisor upgrade since that hadn't been rolled out yet.